### ISSUE TITLE
[question] [enhancement] Will support uploading posts?

### ISSUE THREAD

**Issue by ohaddahan** [social-manager-tools/socialmanagertools-gui#57]
Is there any plan on supporting uploading new posts via the bot?

**Comment by ptkdev** [comment]
This bot work with scraping instagram web site (desktop). All you can do on the website manually can be implemented
- Is possibile upload photo via instagram web site?
- No...

Then this feature is impossibile now. Is a technology limit.

**Comment by ohaddahan** [comment]
Changing the user agent provides the mobile interface, including the upload button.
The issues I had with it, that https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#elementhandleuploadfilefilepaths doesn't work on it.
(works on changing profile pic though)

**Comment by ptkdev** [comment]
Yes, upload profile pic is possibile. Upload new post/stories no :(

**Comment by ptkdev** [comment]
Reopen, in nexts release i try improve this.

**Comment by Cam-B** [comment]
Uploading (scheduled) posts would be a great addition to this tool. Thank you very much for providing this very useful software
